Snakefile

```diff
@@ -227,6 +227,7 @@ rule build_shapes:
         build_shape_options=config["build_shape_options"],
         crs=config["crs"],
         countries=config["countries"],
+        subregion=config.get("subregion",{}),
```

Comment:
we can include "subregion" in the config.default.yaml so we can safely use config["subregion"]